run.sh

```diff
@@ -22,4 +22,6 @@ else
 fi
 
 cd ..
+python manage.py migrate && python install.py
+python manage.py createsuperuser --no-input --username admin --email admin@example.com
```

Comment:
@silentninja Please note that is not idempotent and will fail if you've run it already; we should handle this somehow. I think `install.py` is fixed now, but if it will error we should handle that too.

Reply:
>  Please note that is not idempotent and will fail if you've run it already

The command would throw an error but it wouldn't crash the script. But in long run, we should definitely move it to a migration file and create a `superuser` as part of the migration step. I was planning to create a separate issue for it and fix it along with #202

Reply:
`install.py` should work similarly to Django migration once #1481 is fixed

Reply:
Please do make issues to track these things so we don't forget about them. As long as it doesn't crash the script, that's fine with me.